### PR TITLE
Remove broken test configuration file

### PR DESCRIPTION
**Pull request by oyvindeide**
This points to several paths that are not available and does not work.

## Pre review checklist

- [x] Added appropriate labels

Adding labels helps the maintainers when writing release notes, see sections and the
corresponding labels here: https://github.com/equinor/ert/blob/main/.github/release.yml

### CODE REVIEW

**Comment by oyvindeide** [comment]
test this please

**Comment by oyvindeide** [comment]
test this please

**Comment by oyvindeide** [comment]
test this please

File: tests/libres_tests/res/config/test_config.py [hunk 1]
```diff
@@ -117,14 +117,23 @@ def test_item_types(self):
             self.assertIn("ConfigParser", repr(conf))
             self.assertIn("size=1", repr(conf))
 
+    @tmpdir(None)
     def test_parse(self):
+        config_file = """
+RSH_HOST be-lx655082:2 be-lx633214:2
```
Comment (markusdregi) [inline tests/libres_tests/res/config/test_config.py:123]: Should we replace the machine names by obvious dummy names?

**Review by markusdregi — APPROVED** [review]
Could you add some context in the PR title going into the release notes? Something like:

```
Remove broken test configuration file
```

Besides that and the minor RSH comment, I think this looks good 👌🏻

**Pull request merged**